### ISSUE TITLE
Error when installing JetBrains Mono font via terminal in Linux

### ISSUE THREAD

**Issue by hazem94** [JetBrains/JetBrainsMono#405]
I used the official script by `JetBrains` [link](https://raw.githubusercontent.com/JetBrains/JetBrainsMono/master/install_manual.sh) however, I am getting an error:
```
Downloading latest release info: https://api.github.com/repos/JetBrains/JetBrainsMono/releases/latest
Latest fonts version: v2.225
Downloading fonts archive: https://github.com/JetBrains/JetBrainsMono/releases/download/v2.225/JetBrainsMono-2.225.zip
Extracting fonts: /root/.local/share/fonts
Building fonts cache...
Cleaning up...
unlink: cannot unlink '/tmp/tmp.pjvbTIbzju /tmp/tmp.2wuWWqJPqa': No such file or directory
Unable to unlink: /tmp/tmp.pjvbTIbzju /tmp/tmp.2wuWWqJPqa
```
OS: Ubuntu 20.04.2 LTS through WSL2

**Comment by hznuksco** [comment]
> Installing latest JetBrains Mono fonts...
Checking requirements...
awk... Found
curl... Found
fc-cache... Found
mkdir... Found
mktemp... Found
unlink... Found
unzip... Found
Downloading latest release info: https://api.github.com/repos/JetBrains/JetBrainsMono/releases/latest
Latest fonts version: v2.225
Downloading fonts archive: https://github.com/JetBrains/JetBrainsMono/releases/download/v2.225/JetBrainsMono-2.225.zip
Extracting fonts: /root/.local/share/fonts
Building fonts cache...
Cleaning up...
unlink: cannot unlink '/tmp/tmp.JDpar3EsHF /tmp/tmp.lH9MAhQ2jA': No such file or directory
Unable to unlink: /tmp/tmp.JDpar3EsHF /tmp/tmp.lH9MAhQ2jA

OS: Ubuntu 18.04.5 LTS x86_64

**Comment by philippnurullin** [comment]
Hi @hazem94 . The script for installing on Linux was added by @nisrulz . I believe it was added by this PR https://github.com/JetBrains/JetBrainsMono/pull/288

**Comment by nisrulz** [comment]
Yes. I can probably help for the issue on Linux in general, but I am not sure about the WSL version. I am not aware of using the WSL. 

I will look into the issue and provide a fix.

**Comment by nisrulz** [comment]
@philippnurullin The script I provided initially has been modified and is being worked on by others already. 

![Screenshot 2021-02-04 at 12 34 00](https://user-images.githubusercontent.com/2096087/106887534-9186b300-66e5-11eb-869c-b32ee6763be3.png)

I can't find any info regarding @manlix and @wolkenschieber, but I am guessing since they are Jetbrain employees because there commits didn't come from PR but rather are direct pushes to the repo. 

I guess they are in better position to handle this issue, since the script has been modified to enable font-caching which was added by @manlix and is the issue here.

**Comment by nisrulz** [comment]
If they don't pick it up, I will take a look nevertheless 😅  Should be fixable.

**Comment by philippnurullin** [comment]
@nisrulz Thanks for the lightning speed reply.

**Comment by manlix** [comment]
@philippnurullin  I think it's a bug from PR #402 =)

**Comment by manlix** [comment]
@philippnurullin Could you merge fix #406 to fix the issue?

**Comment by philippnurullin** [comment]
@manlix Thanks for the quick response.
